src/integrated/soil_snow_model.jl

```diff
@@ -253,7 +253,7 @@ function update_soil_snow_ground_heat_flux!(
     κ_soil = ClimaLand.Domains.top_center_to_surface(p.soil.κ)
 
     # Depth of snow and soil layers interacting thermally at interface
-    Δz_snow = p.snow.z # Snow depth
+    Δz_snow = Snow.snow_depth(snow_params.density, Y, p, snow_params) #Snow depth

```

Comment:
I am wondering why we cannot do the formula Y.snow.S * p.snow.rho_snow / rho_liq here

Reply:
The two are identical for the given configuration, it only changes if the type of the density model changes.